netkan/netkan/cli.py

```diff
@@ -105,18 +105,18 @@ def indexer(queue, metadata, token, repo, user, key,
     '--schedule-all', is_flag=True, default=False,
     help='Schedule all modules even if we think they should be covered by webhooks'
 )
-def scheduler(queue, netkan, max_queued, debug, dev, schedule_all):
+def scheduler(queue, netkan_remote, max_queued, debug, dev, schedule_all):
     level = logging.DEBUG if debug else logging.INFO
```

Comment:
Option is still `--netkan`, so the named param won't match

Reply:
Ahh, I thought these were matched up by position, and I didn't want to break any existing command line usages. But apparently we can add a new name without removing the old one:

https://click.palletsprojects.com/en/7.x/options/

Reply:
Fixed, now it works with either `--netkan` or `--netkan-remote`.